console.log("Hey")
try{
    console.log(hey)
}
catch(err){
    console.log(err)
}

console.log("Hey")